splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -92,7 +92,7 @@ private void configureErrorHandler() throws Exception {
             .setHeader("outcome-fail", simple("true"))
             .process(resultTransformer)
             .marshal().json()
-            .log("Fail with for id ${header.ce-id} : ${exception.message}")
```

Comment:
Unrelated to current change, but wanted to ask whether the message id is present in the log? Where you able to test out the message?